README.rst

```diff
@@ -35,26 +35,30 @@
 .. |Tracking| image:: https://img.shields.io/badge/issue_tracking-github-blue
         :target: https://github.com/Billingegroup/scikit-package/issues
 
-``scikit-package`` is a Python package standard and generator for scientific code. Use ``scikit-package`` to launch a new project or migrate existing ones to support the latest Python versions and streamline the process of distributing and maintaining your software package.
+``scikit-package`` offers tools and practices for the scientific community to make better and more reusable Scientific Python packages and applications:
 
-- Automated `PEP8 <https://peps.python.org/pep-0008/>`_ and `PEP256 <https://peps.python.org/pep-0256/>`_ standard checks.
```

Comment:
This was copied from the official doc.. but I think it's a bit too technical for readme since now our audience is much broader based on the new story.